src/Avalonia.Controls.TreeDataGrid/Models/TreeDataGrid/ITextCellOptions.cs

```diff
@@ -1,9 +1,21 @@
-﻿using Avalonia.Media;
+﻿using System.Globalization;
+
+using Avalonia.Media;
 
 namespace Avalonia.Controls.Models.TreeDataGrid
 {
     public interface ITextCellOptions : ICellOptions
     {
+        /// <summary>
+        /// Gets the format string to be used to format the cell value.
+        /// </summary>
+        string StringFormat { get; }
```

Comment:
I guess this won't be possible since ITextCellOptions is not generic and passing in the type of the property for the column but I thought I'd bring it up just in case you see a way to make it happen!
It would be nice to decorate the property with `[StringSyntax(StringSyntaxAttribute.NumericFormat)]` or any of the other derivatives.

Again, I don't think this would work HERE but it would be cool to be able to specify the stringsyntax somehow.

Reply:
Of course! I was thinking of having it set according to the TModel type, but you're right that the possibilities are so varied, I don't think it's worth going through all this trouble.